content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,57 @@
+
+ - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
+Under the helptext there are 5 fields that can be pressed, and when pressed on any of the selections will be added to the address format input box:
+     - **Customer** -  in this section it's possible to press the "+" sign to add to the format the following inputs: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
+     - **Warehouse** - in this section it's possible to press the "+" sign to add to the format the following inputs: reference, name, management_type, htmlFields
+     - **Country** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, zip_code_format, htmlFields
+     - **State** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, htmlFields 
+     - **Address** - in this section it's possible to press the "+" sign to add to the format the following inputs: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
+     
+ Below there are 4 buttons with different actions:
+ 
+ 
+     - **Use the last registered format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the last registered formats.
+     - **Use the default format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the default address formats.
+     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will input the address format box with the current modified formats.
+     - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
+When pressed "ok" will remove all the formats.
+     
+ - **Active** - Toggle switch that can be turned to "Yes" or "No". By default is set to "yes".
+ - **Contains states** - Toggle switch that can be turned to "Yes" or "No". By default is set to "no".
```

Comment:
```suggestion
 - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default is set to "no". If set on "yes", this adds a new field to the address format.
```